src/test/java/com/softserveinc/ita/rozetka_test/RozetkaProductNavigationTest.java

```diff
@@ -53,13 +75,63 @@ public void testResultsOnProductsListPage() {
                 .openProductsListPage(productCategoryName);
         assertEquals(productListPage.getPageTitle(), productCategoryName, format("Title should be '%s'", productCategoryName));
         var expectedProductType = "Ноутбук";
-        assertTrue(productListPage.getProductNameForProduct(1).contains(expectedProductType),
+        assertTrue(productListPage.getProductName(1).contains(expectedProductType),
                 format("First product should be '%s'", expectedProductType));
         var productCount = productListPage.getProductListSize();
-        assertTrue(productListPage.getProductNameForProduct(productCount / 2).contains(expectedProductType),
+        assertTrue(productListPage.getProductName(productCount / 2).contains(expectedProductType),
                 format("Middle product should be '%s'", expectedProductType));
-        assertTrue(productListPage.getProductNameForProduct(productCount).contains(expectedProductType),
+        assertTrue(productListPage.getProductName(productCount).contains(expectedProductType),
                 format("Last product should be '%s'", expectedProductType));
     }
+
+    @Test
+    public void testSortByPrice() {
+        var productListPage = rozetkaHomePage
+                .getHeader()
+                .searchFor("сокира")
+                .setSortingType(SortingOption.CHEAP);
+        var productsAmount = productListPage.getProductListSize();
+        var firstProductPrice = productListPage.getProductPrice(1);
+        var lastProductPrice = productListPage.getProductPrice(productsAmount);
+        var middleProductPrice = productListPage.getProductPrice(productsAmount / 2);
+        assertTrue(middleProductPrice < lastProductPrice && firstProductPrice < middleProductPrice,
```

Comment:
do not combine asserts, when it fails, you won't be able to tell what exactly failed